### DISCUSSION TITLE
How can I execute files to test my code?

### DISCUSSION THREAD

**Discussion by DaeyongKwon98** [ANLAB-KAIST/KENSv3#97] [Project Q&A 1. Application Layer (Socket)] [answered]
I tried to compile .cpp file with $ gcc NAME.cpp, 
then following error occurs.

In file included from EchoAssignment.cpp:1:
EchoAssignment.hpp:6:10: fatal error: E/E_Common.hpp: No such file or directory
    6 | #include <E/E_Common.hpp>
      |          ^~~~~~~~~~~~~~~~

Which file I should compile, and how to do it?
Sorry for such a silly question :(
Thank you.

**Comment by whkim0** [comment, marked as answer]
You can find your answers in "Getting Started" ([https://github.com/ANLAB-KAIST/KENSv3/wiki#getting-started](https://github.com/ANLAB-KAIST/KENSv3/wiki#getting-started)) in KENSv3 Wiki. 😃
